src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -142,55 +141,54 @@ func (s *Service) cmdGithubCreate(roomID, userID string, args []string) (interfa
 	return gomatrix.TextMessage{"m.notice", fmt.Sprintf("Created issue: %s", *issue.HTMLURL)}, nil
 }
 
-func (s *Service) cmdGithubComment(roomID, userID string, args []string) (interface{}, error) {
+const cmdGithubReactUsage = `!github react [owner/repo]#issue (+1|-1|laugh|confused|heart|hooray)`
```

Comment:
(Plus it is what it refers to them in the UI)
![image](https://user-images.githubusercontent.com/2403652/29617246-311a4862-880c-11e7-8694-557be0121b16.png)